libraries/apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/Options.kt

```diff
@@ -143,7 +140,6 @@ class CodegenSchemaOptions(
     val packageName: String? = null
 )
 
-@ApolloExperimental
```

Comment:
These are all the sam kind of symbols, make `ApolloCompiler` &`FooOptions` have all the same visibility